plugins/modules/route53_health_check.py

```diff
@@ -577,6 +618,20 @@ def main():
     health_check_name = module.params.get("health_check_name")
     tags = module.params.get("tags")
     purge_tags = module.params.get("purge_tags")
+    child_health_checks_in = module.params.get("child_health_checks")
+    health_threshold_in = module.params.get("health_threshold")
+
+    # Throw an exception if the input parameters for a calculated health check are invalid
```

Comment:
Thank you for the feedback. I am slightly confuse on how to implement `mutually_exclusive`.

Reply:
Since `child_health_checks` is required if `type` is CALCULATED, I think you can add something like this to`args_mutually_exclusive  in line 592:
```
["child_health_checks", "ip_address"],
["child_health_checks", "port"],
["child_health_checks", "fqdn"],
```

Reply:
Oh, I get it now. Thank you. I will change it.